fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -887,6 +887,7 @@ IKeyValueStore* keyValueStoreMemory(std::string const& basename,
 	    .detail("MemoryLimit", memoryLimit)
 	    .detail("StoreType", storeType);
 
+	// SOMEDAY: Maybe update to use DiskQueueVersion::V2 with xxhash3 checksum.
```

Comment:
The transaction state store will use `KeyValueStoreMemory`, so even if we're not using the `memory` storage engine, I think we should use `DiskQueueVersion::V2` here. Is there a reason not to make that change in this PR?

Reply:
I'm new to the code and was not sure if it will cause problem if existing user upgrade from disk queue version V0 or V1 . Discussed with @sfc-gh-satherton offline and seems it's fine. I will change it to V2.

Reply:
Actually since we are not going to patch 7.0 for the change, I will leave `KeyValueStoreMemory` to use V1 and change it later in FDB 7.2.